release-notes.md

```diff
@@ -1,9 +1,17 @@
 # Release notes
 
-## October 3, 2019
+## October 4, 2019
 
 The following updates were made in this release:
 
+**iOS Core 2.3.4**
+* Fixed a crash that may happend during app shutdown.
+* Fixed a bug where SDK does not functioning in the race condition when SDK is being initiliazed in multiple thread.
```

Comment:
Please reword to:
"Fixed a bug where,  when SDK is being used in multiple threads, the SDK does not function in the race condition.